packages/server/test/unit/gui/auth_spec.js

```diff
@@ -123,38 +122,24 @@ describe('lib/gui/auth', function () {
   })
 
   context('.start', () => {
-    it('focuses main window upon successful auth', async () => {
+    it('resolves upon successful auth', async () => {
       sinon.stub(user, 'getBaseLoginUrl').resolves('www.foo.bar')
       sinon.stub(Promise, 'fromCallback').resolves()
       sinon.stub(auth._internal, 'launchServer').resolves()
       sinon.stub(auth._internal, 'buildLoginRedirectUrl').resolves('www.redirect.url')
       sinon.stub(auth._internal, 'launchNativeAuth').resolves()
       sinon.stub(auth._internal, 'stopServer')
-      sinon.stub(windows, 'focusMainWindow').callsFake(() => {})
 
-      await auth.start(() => {}, 'code', () => {
-        windows.focusMainWindow()
-      })
+      await auth.start(() => {}, 'code')
 
       expect(auth._internal.stopServer).to.be.calledOnce
-      expect(windows.focusMainWindow).to.be.calledOnce
     })
 
-    it('focuses main window when auth fails', async () => {
+    it('resolves when auth fails', async () => {
       sinon.stub(user, 'getBaseLoginUrl').rejects(new Error('test error'))
       sinon.stub(auth._internal, 'stopServer')
-      sinon.stub(windows, 'focusMainWindow').callsFake(() => {})
 
-      try {
-        await auth.start(() => {}, 'code', () => {
-          windows.focusMainWindow()
-        })
-      } catch (e) {
-        expect(e.message).to.eql('test error')
-      }
```

Comment:
this `auth.start` call has actually never rejected - `auth.start` errors are propagated via `onMessage`. so this `catch` block never fired.